pkgs/applications/misc/parsec/default.nix

```diff
@@ -0,0 +1,64 @@
+{ stdenv
+, lib
+, buildFHSUserEnv
+, parsecDrv ? null
+, dpkg
+, fetchurl
+}:
+
+let
+  parsecDrvUsed = if parsecDrv != null then parsecDrv else stdenv.mkDerivation rec {
+    name = "parsec-extracted";
+    src = fetchurl {
+      url = "https://builds.parsecgaming.com/package/parsec-linux.deb"; # Can't seem to find a link to a specific version, hopefully this wont change invalidating the derivation.
```

Comment:
https://aur.archlinux.org/packages/parsec-bin

I think it is fair to use the versioned version (even if it is an older build) to ensure reproducibility here.

Reply:
See also the fixed comment from AUR.